app/config/config.yml

```diff
@@ -237,12 +237,18 @@ scheb_two_factor:
         cookie_lifetime: 2592000
 
     email:
-        enabled: "%twofactor_auth%"
+        enabled: false
         sender_email: "%twofactor_sender%"
         digits: 6
         template: WallabagUserBundle:Authentication:form.html.twig
         mailer: wallabag_user.auth_code_mailer
 
+    google:
+        enabled: true
+        server_name: wallabag
```

Comment:
Regarding #3173 I was wondering if the Google auth needs a _domain name_ with or without the protocol?